app/models/mongo_clinic.rb

```diff
@@ -0,0 +1,82 @@
+# Object representing a clinic that a patient is going to.
+class MongoClinic
```

Comment:
Copying this over into its own deal so we can still access the collection in porting scripts, as we have previously.